force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -49,8 +49,18 @@ public with sharing class ClassesObjectsMethods {
         // In the first position add the string length
         // In the second position add the substring
         // In the third position add the string in lower case
+        String str = 'Hello, Salesforce!';
 
-        return null; // Replace null with the variable you used to store the result
+        Integer strLength = str.length();
+        String strSubString = str.substring(0, 5);
+        String strLower = str.toLowerCase();
+
+        List<String> results = new List<String>();
+        results.add(String.valueOf(strLength));
+        results.add(strSubString);
+        results.add(strLower);
+        
+        return results; 
```

Comment:
Good solution. I would say that to be more concise you can add the method calls directly to the List and not sacrifice any readability. 

My solution is below. 
```
 public static List<String> practiceStringMethods() {
        List<String> strings = new List<String>();
        String helloSalesforce = 'Hello, Salesforce!';
        strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
    }
    ```